addons/crm/models/crm_lead.py

```diff
@@ -1231,14 +1231,9 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
-            ('alias_name', '!=', False),
-            ('alias_name', '!=', ''),
-            ('alias_model_id.model', '=', 'crm.lead'),
-            ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
-        ], limit=1)
-        if alias_record and alias_record.alias_domain and alias_record.alias_name:
+        sales_team = self.env.user.sale_team_id
+        alias_record = sales_team.alias_id
+        if sales_team.active and all([alias_record, alias_record.alias_domain, alias_record.alias_name]):
```

Comment:
it was already there but you can not put the alias_record, on an empty record alias_record.alias_domain and alias_record.alias_name will both return False